doc/topics/windows/windows-package-manager.rst

```diff
@@ -770,34 +767,38 @@ Firefox:
         uninstaller: '%ProgramFiles(x86)%/Mozilla Firefox/uninstall/helper.exe'
         uninstall_flags: '/S'
 
-You can see the first item is the short name for the software, in this case
-``firefox_x64``. It is the first line in the definition. The next line is
-indented two spaces and contains the software ``version``. The lines following
-the ``version`` are indented two more spaces and contain all the information
-needed to install the Firefox package.
+The first line is the short name of the software which is ``firefox_x64``.
+.. important::
+    The short name name must match exactly what is shown in Add/Remove Programs (``appwiz.cpl``)
+    and it must be unique across all other short names in the software repository.
+    The ``full_name`` combined with the version must also be unique.
 
+The second line is the ``software version`` and is indented two spaces.
 .. important::
-    The package name must be unique to all other packages in the software
-    repository. The ``full_name`` combined with the version must also be unique.
-    They must also match exactly what is shown in Add/Remove Programs
-    (``appwiz.cpl``).
+    The version number must be enclosed in quotes else the YAML parser removes the trailing zeros.
+    For example, if the version number 74.0 is not enclosed within quotes then the version number
+    is considered as only 74.
+
+The lines following the ``version`` are indented two more spaces and contain all the information
+needed to install the Firefox package.
 
 .. important::
-    The version number must be enclosed in quotes, otherwise the YAML parser
-    will remove trailing zeros. For example, `74.0` will just become `74`.
+    You can specify multiple versions for a software by specifying multiple version numbers at
+    the same indentation level as the first with its software definition below it.
 
-As you can see in the example above, a software definition file can define
-multiple versions for the same piece of software. These are denoted by putting
-the next version number at the same indentation level as the first with its
-software definition information indented below it.
+Example: JINJA templated package definition file
+=================================================
+JINJA is the default templating language used in package definition files.
+You can use JINJA to add variables, expressions to package definition files
+that get replaced with values when the ``.sls`` go through Salt renderer.
 
-Example: Jinja
-==============
+When there are tens or hundreds of versions available for a piece of software the
+definition file can become large and cumbersome to write .
```

Comment:
**Nitpick (non-blocking):** It seems like there's an extra space after "write" here.

Reply:
Updated